src/components/formik-forms/formik-input.jsx

```diff
@@ -11,9 +11,10 @@ require('../forms/row.scss');
 const FormikInput = ({
     className,
     error,
+    validationClassName,
     ...props
 }) => (
-    <div className="col-sm-9 row">
+    <div className="col-sm-9 row row-relative">
```

Comment:
What requires the row to be relative? Is it because it might have a validation message? Right now it's not clear to me from this why I would need this class or not if I were extending this or re-using it in the future. Can the class name refer to why it's needed (e.g. `has-validation` or something specific like that) so that's self-documenting?

Reply:
Renamed -- LMK what you think.